### ISSUE TITLE
Contribute runtime files to vim

### ISSUE THREAD

**Issue by flwyd** [elixir-editors/vim-elixir#565]
The feature wishlist in the README includes "Regularly merging vim-elixir into vim and keeping the sync up-to-date" but I don't see an issue tracking it.

In the absence of a regular process for doing this, would it be possible to do it once?  https://github.com/vim/vim currently has filetype detection and basic `ftplugin` config for `elixir` and `heex` (but not `eelixir).  It doesn't have any syntax or indent files for the language, so editing Elixir files with a standard vim install is a pretty spartan experience.

**Comment by jbodah** [comment]
I agree that this is something we should do. I am trying to be 100% hands off on this project at the moment and have reached out to several potential people who may be interested in helping takeover maintenance this project. This feels like a good first task.

**Comment by samanera** [comment]
In the case of Vim, the discussion to add `HEEx` support happened on a pull request: https://github.com/vim/vim/pull/11169 (it is forwarded automatically to the `vim-dev` maillist, so it reduces the entry barrier).

I see some similarities between `vim-ruby` and `vim-elixir`.
The syntax file included with `$VIMRUNTIME` for [`eruby`](https://github.com/vim/vim/blob/master/runtime/syntax/eruby.vim) is identical to the one provided by [`vim-ruby/eruby`](https://github.com/vim-ruby/vim-ruby/blob/master/syntax/eruby.vim) (except for a comment mentioning who is the _release coordinator_).
Same story with the rest of the files (`ftplugin`, `ident`, `compiler`, ...).

Updates happen from time to time to sync the `$VIMRUNTIME` with the `vim-ruby` repository: https://github.com/vim/vim/commit/da16a1b471aa717f58909cc6531cb6dbbff14d22.

Sometimes, the `neovim` community cherry-pick patches from `vim` to integrate them into their runtime: https://github.com/neovim/neovim/commit/61ccdb2db62481a38604426be530753ad2f9e7cb.

With all this context, I think the process would envolve designating a _release-coordinator_ and submitting the necessary files.

Hope this comment shed some light on this issue. 🙂
